packages/cozy-stack-client/src/urls.js

```diff
@@ -0,0 +1,42 @@
+/* global URL */
```

Comment:
URL is not globally available in nodeJS : `ReferenceError: URL is not defined`. Is it injected via webpack ?

Reply:
You mean injected by the app?

Reply:
What about checking the existence of `URL` and using something like https://www.npmjs.com/package/whatwg-url when it doesn't exist?

Reply:
I import url [here](https://github.com/cozy/cozy-client/pull/331/files#diff-4420294e0c84ddd3b402e362c16e2662R36).